src/components/MainPageComponent.tsx

```diff
@@ -1,7 +1,23 @@
-import { Container, Typography, useTheme } from '@mui/material';
+import {
+  Avatar,
+  Box,
+  Container,
+  Grid,
+  Typography,
+  useTheme,
+} from '@mui/material';
 
 function MainPageComponent() {
   const theme = useTheme();
+  const boxSx = {
+    border: '4px solid',
+    borderColor: theme.palette.primary.main,
+    backgroundColor: theme.palette.secondary.main,
+    borderRadius: '4px',
+    padding: '1rem',
+    mb: '1rem',
+  };
+
```

Comment:
I think we need styled box with this sx as separate component, except margin bottom of course, and padding maybe.

Reply:
> I think we need styled box with this sx as separate component, except margin bottom of course, and padding maybe.

Yes, we do